docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -78,7 +78,7 @@ Both Hugo and Astro offer built-in support for building, bundling and minifying
 
 Jekyll uses an older [templating language](https://jekyllrb.com/docs/liquid/) to render your website called Liquid. Astro lets you create pages using your favorite UI component libraries (React, Preact, Vue, Svelte, and others) or a built-in component syntax that is similar to HTML/JSX. Jekyll does not support using modern UI components for HTML templating.
 
-#### Comparing Jekyll vs. Astro Performance
+### Comparing Jekyll vs. Astro Performance
```

Comment:
these are supposed to be `####`

Reply:
Ok, maybe we can add some custom CSS for this page that makes those smaller? From what I understand headers are required to increase by only one at a time, and we're meant to use CSS to make the size the size we want.

Reply:
Ah gotcha. We use `h2` + `h3` to go in the sidebar, so I'd really like to keep them as h4 if possible since they don't deserve a spot in our sidebar.

That feels more like an SEO thing than an accessibility thing, I'd love to learn more about the reasoning behind this from an a11y perspective!

Reply:
Here's the accessibility rule from axe: [rule - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/4.3/heading-order?application=AxeFirefox)

Reply:
I'll revert this change for now.